tests/unit/test_parser.py

```diff
@@ -101,6 +100,13 @@ def test_parser_rhs_index_shift_multiple():
     assert statement.__str__() == expected.__str__()
 
 
+def test_parser_invalid_statement():
+    with pytest.raises(Exception):
```

Comment:
Should match part of the error string of this exception (to make sure the expected exception is actually being thrown).

An example from https://docs.pytest.org/en/6.2.x/assert.html is:

```python
def test_match():
    with pytest.raises(ValueError, match=r".* 123 .*"):
        myfunc()
```